### DISCUSSION TITLE
Setup instructions without all the bolierplate

### DISCUSSION THREAD

**Discussion by TheKnarf** [remix-run/remix#2827] [Proposals]
I can't seem to find instructions on how to setup Remix manually without using a template like `remix-run/indie-stack`. I'd much rather have a minimal setup without Prisma, tailwindCSS, Cypress and all the other boilerplate that template brings inn.

I want something similar to [Next.js manual setup guide](https://nextjs.org/docs/getting-started#manual-setup).

Seems that `npx create-remix@latest` can give me a smaller setup without all the boilerplate, but I still can't find a total manual setup instructions.

**Comment by sergiodxa** [comment]
When you create a new app using the CLI you need to choose between a stack or using just the basics.

**Reply by TheKnarf** [reply]
I would still like some documentation on how to set it up manually without the CLI. Similarly to the way Next.js does it.
